### PR TITLE
Added Cypher Page and Cypher List Page

### LINKED ISSUE #3: Cypher Builder Page
Create a cypher builder page as well as a Cyphers admin page to display all cyphers to manage or add new ones.

### PR DESCRIPTION
**Pull request by herrozerro**
Added Cypher Builder

Closes #3

### CODE REVIEW

File: src/CypherTools.Web/Client/Pages/Cypher.razor [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    <div>
+        <label>
+            Source:
+            <input @bind="cypher.Source" />
+        </label>
+    </div>
+}
+
+
+@code {
```
Comment (williamkeller) [inline src/CypherTools.Web/Client/Pages/Cypher.razor:104]: Ewwwww.....code in the template page! It's like old asp all over again.  🤮

File: src/CypherTools.Web/Client/Pages/Cypher.razor [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            }
+            catch (AccessTokenNotAvailableException exception)
+            {
+                exception.Redirect();
+            }
+
+        }
+    }
+
+    private async Task SaveCypher()
```
Comment (williamkeller) [inline src/CypherTools.Web/Client/Pages/Cypher.razor:132]: On a more serious note, why is this here instead of in the controller? Is that a razor thing?

**Review by williamkeller — APPROVED** [review]
Couple comments related to razor, since I don't know anything about that, but I trust you know what you're doing.

File: src/CypherTools.Web/Client/Pages/Cypher.razor (hunk 2 above)
Comment (herrozerro) [reply src/CypherTools.Web/Client/Pages/Cypher.razor:132]: The controller code has the actual database work.  This is just the pages code to get and save the objects.
